src/modern/fast-csv-wrapper.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash/fp'
+import { format as formatCSV } from '@fast-csv/format'
```

Comment:
This file can hopefully go away once https://github.com/C2FO/fast-csv/issues/565 and https://github.com/C2FO/fast-csv/issues/566 are resolved